### PR TITLE
catch a corner case for elliptic surfaces

### PR DESCRIPTION
**Pull request by simonbrandhorst**

### CODE REVIEW

File: experimental/Schemes/src/elliptic_surface.jl [hunk 1]
```diff
@@ -2296,11 +2314,13 @@ function _elliptic_surface_with_trafo(g::MPolyRingElem{<:AbstractAlgebra.Generic
 
   # The following three commands won't work unless we convert to a rational_function_field
   EE = base_change(x->evaluate(x, t), E)
-
-  EE = tates_algorithm_global(EE)
-  EE, _ = short_weierstrass_model(EE)
-  EE, _ = integral_model(EE)
-
+  @show EE
```
Comment (lgoettgens) [inline experimental/Schemes/src/elliptic_surface.jl]:
```suggestion
```
just a reminder to remove this `show` before merging

Comment (simonbrandhorst) [reply experimental/Schemes/src/elliptic_surface.jl]: Thank you! Got it.

**Review by lgoettgens — COMMENTED** [review]
I am not familiar with the involved mathematics here, so I do not see myself appropriate to approve

**Comment by simonbrandhorst** [comment]
You have my word, that it is okay. (nothing deep anyways). If you still feel uneasy, I can ask someone else :-).

**Comment by HechtiDerLachs** [comment]
@simonbrandhorst : It looks like we need this patch for the current version of our notebook to actually run. Is there a reason you removed the backport label back then?

**Comment by simonbrandhorst** [comment]
I thought it was too late to backport it. Would be happy if it was backported.

**Pull request merged**